model/code/registry.yaml

```diff
@@ -7,38 +7,38 @@ groups:
     attributes:
       - id: code.function
         type: string
-        stability: experimental
+        stability: experimental # RELEASE CANDIDATE
         brief: >
           The method or function name, or equivalent (usually rightmost part of the code unit's name).
         examples: serveRequest
       - id: code.namespace
         type: string
-        stability: experimental
+        stability: experimental # RELEASE CANDIDATE
```

Comment:
let's add this to the notes also (similar to #1101), so that it's visible in the markdown as well:

> This attribute has stability level RELEASE CANDIDATE.